src/test/java/io/vertx/test/core/FileSystemTest.java

```diff
@@ -538,6 +539,8 @@ public void testChownToOwnUser() throws Exception {
 
   @Test
   public void testChownToOwnGroup() throws Exception {
+    // Not supported in WindowsFileSystemProvider
+    Assume.assumeFalse(System.getProperty("os.name").startsWith( "Windows" ));
```

Comment:
Maybe create a method isWindows() containing:
System.getProperty("os.name").startsWith( "Windows" )
to avoid writing the same code many times